sno/fast_import.py

```diff
@@ -22,10 +22,10 @@ def fast_import_tables(
     quiet=False,
     header=None,
     message=None,
+    replace_existing=False,
```

Comment:
Let's merge "replace_existing" and "incremental" into a single three-state flag - something like:
start_from
replace_existing= one of:
REPLACE_NONE,
REPLACE_GIVEN,
REPLACE_ALL,